package.json

```
@@ -35,7 +35,7 @@
     "extract-changelog": "rm -rf NEXT_RELEASE_CHANGELOG.md && yarn lerna-workspaces run extract-changelog",
     "bootstrap": "yarn lerna-workspaces run bootstrap",
     "bootstrap-ci": "yarn lerna-workspaces run bootstrap-ci",
-    "lint": "yarn lerna-workspaces run lint",
+    "lint": "yarn lerna-workspaces run lint && cd docusaurus && npx prettier --write '**/*.mdx'",
```

```suggestion
    "lint": "yarn lerna-workspaces run lint && npx prettier --check 'docusaurus/**/*.mdx'",
    "lint-fix": "yarn lerna-workspaces run lint-fix && npx prettier --write 'docusaurus/**/*.mdx'",
```
^ Makes it more consistent with the command in package/package.json

---

But here the purpose is to format the documentation markdown files so that is why we should have a `write` flag. The markdown files should be formatted when we run the lint command.

---

I have changed the lint command for the check_pr workflow file to `yarn lerna-workspaces run lint && cd docusaurus && npx prettier --check '**/*.mdx` and now I think it will make sense.

---

In `package/package.json` we have one command to lint files, and one to autofix them with prettier. (`lint` and `lint-fix`).

Why does it make sense to make this different here? I don't agree that this is better.

---

The reason is that the docusaurus is not exposed to the package directory so keeping the command in the root makes more sense according to me. We can have a docs:lint-fix command on root package.json to deal with it.

---

> the docusaurus is not exposed to the package directory

I don't understand how that affects anything here, this command is run from the root and not `package/`. Lerna will in this case just run `yarn run lint` in `package`. Why would it only lint there, but write files in place here? It just seems confusing to anybody running these things for the first time.

Wanna jump on a quick call to reach some mutual understanding/agreement here? cc @vishalnarkhede in case you'd like to join as well